src/js/components/SelectMultiple/SelectionSummary.js

```diff
@@ -68,7 +68,7 @@ const SelectionSummary = ({
     value?.length === 0 ||
     selectedValuesDisabled() ||
     !value ||
-    selectedInSearch().length === 0
+    (!limit && selectedInSearch().length === 0)
```

Comment:
Do we still need this change? I didn't see a difference in behavior with vs without it

Reply:
good catch yes we dont need